tests/integration_tests.rs

```diff
@@ -193,9 +198,20 @@ async fn test_wasi_provider() -> Result<(), Box<dyn std::error::Error>> {
     let mut logs = pods.log_stream("hello-wasi", &LogParams::default()).await?;
 
     while let Some(line) = logs.try_next().await? {
-        assert_eq!("Hello World!\n", String::from_utf8_lossy(&line));
+        assert_eq!("Hello, world!\n", String::from_utf8_lossy(&line));
     }
 
+    let pod = pods.get("hello-wasi").await?;
+
+    let state = pod.status.unwrap().container_statuses.unwrap()[0]
```

Comment:
This is super gnarly. I know that this is a test but can we clean this up?

Reply:
Have a way of doing it that doesn't involve another function? I tried a couple different things and none of them worked